l3experimental/l3draw/l3draw.dtx

```diff
@@ -420,7 +420,7 @@
 % higher-level functions, for example path construction, and allows parts of
 % a drawing to be rotated, scaled or skewed. This occurs before writing any
 % data to the driver, and so such manipulations are tracked by the drawing
-% mechanisms. See \cs{@@_backend_cm:nnnn} for backend-level
+% mechanisms. See \cs{__draw_backend_cm:nnnn} for backend-level
```

Comment:
Stray edit?

Reply:
To avoid `@@` shown in `l3draw` doc (2024-02-20), p. 7.
![image](https://github.com/latex3/latex3/assets/6376638/51cad4b3-797d-4101-b445-3337e95e44d8)

Reply:
That should be fixed by adding `@@` setup to `l3draw.dtx` ...

Reply:
The corresponding commit is now dropped from this PR.

Reply:
If I add
```tex
%    \begin{macrocode}
%<@@=draw>
%    \end{macrocode}
```
right after `\begin{documentation}` in `l3draw.dtx`, there will be a unwanted code line `<@@=draw>` numbered 1 at the beginning of  `l3draw.pdf`.

Checking occurrences of `__` in `interface.pdf`, the module names are actually spelled out in source files.

https://github.com/latex3/latex3/blob/3b901d0e778f9ec93c03daf6d68ee97ab5668a7e/l3kernel/l3regex.dtx#L895-L896
https://github.com/latex3/latex3/blob/3b901d0e778f9ec93c03daf6d68ee97ab5668a7e/l3kernel/l3regex.dtx#L908-L909
https://github.com/latex3/latex3/blob/3b901d0e778f9ec93c03daf6d68ee97ab5668a7e/l3kernel/l3fp.dtx#L1640-L1643

Maybe you know better ways?